src/NRedisStack/Search/FTSpellCheckParams.cs

```diff
@@ -0,0 +1,103 @@
+using NRedisStack.Search.Literals;
+namespace NRedisStack.Search
+{
+    public class FTSpellCheckParams
+    {
+        List<object> args = new List<object>();
+        private List<KeyValuePair<string, string>>? terms = null;
+        private int? distance = null;
+        private int? dialect = null;
+
+        public FTSpellCheckParams() { }
+
+        /// <summary>
+        ///  Specifies an inclusion (INCLUDE) of a custom dictionary.
+        /// </summary>
+        public FTSpellCheckParams IncludeTerm(string dict)
+        {
+            return AddTerm(dict, SearchArgs.INCLUDE);
+        }
+
+        /// <summary>
+        ///  Specifies an inclusion (EXCLUDE) of a custom dictionary.
+        /// </summary>
+        public FTSpellCheckParams ExcludeTerm(string dict)
+        {
+            return AddTerm(dict, SearchArgs.EXCLUDE);
+        }
+
+        /// <summary>
+        ///  Specifies an inclusion (INCLUDE) or exclusion (EXCLUDE) of a custom dictionary.
+        /// </summary>
+        private FTSpellCheckParams AddTerm(string dict, string type)
+        {
+            if (terms == null)
+            {
+                terms = new List<KeyValuePair<string, string>>();
```

Comment:
If this would be default value for `terms` you don't need null check condition here